src/main/java/data/TaskManager.java

```diff
@@ -75,7 +112,7 @@ public static void deleteManager(Scanner scanner, WeekView weekView, TaskManager
     }
 
     // to abstract as Parser/UI function
-    private static LocalDate getStringFromUser (Scanner scanner) throws DateTimeParseException {
+    private static LocalDate getStringFromUser(Scanner scanner) throws DateTimeParseException {
```

Comment:
The name of this method could perhaps be more specific, for example  "parseInputDate" to make it clearer at first glance the objective of this method.